src/app/tasks/modals/grade-task-modal/grade-task-modal.component.html

```diff
@@ -0,0 +1,66 @@
+<mat-card class="grade-task-modal">
+  <mat-card-title> Assess Task Quality </mat-card-title>
+  <mat-card-content>
+    <!-- For Grade Tasks -->
+    <div *ngIf="task.definition.is_graded">
+      <p>
+        Please provide a grade for task
+        <label class="task-label">
+          {{ task.definition.abbreviation }}
+        </label>
+      </p>
+      <div class="grade-toggle-group">
+        <mat-button-toggle-group #group="matButtonToggleGroup" (change)="updateGrade($event.value)">
+          <mat-button-toggle
+            *ngFor="let idx of gradeValues"
+            class="grade-toggle"
+            value="{{ idx }}"
+            aria-label="{{ grades[idx] }}"
+            matTooltip="This student completed the task to a {{ grades[idx] }} standard"
+            [matTooltipShowDelay]="500"
+            aria-describedby="This student completed the task to a {{ grades[idx] }} standard"
+          >
+            <grade-icon [grade]="grades[idx]"></grade-icon>
+          </mat-button-toggle>
+        </mat-button-toggle-group>
+      </div>
+    </div>
+    <br *ngIf="task.definition.is_graded && task.definition.max_quality_pts > 0" />
```

Comment:
I think `<br >` tag is not a good practice for creating space between elements. Check this [threat](https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br). 

I recommend to use CSS margin with [ngStyle](https://angular.io/api/common/NgStyle).

Reply:
I think that's another great point. Thanks for sharing the stack overflow link, I will fix this up too with ngStyle. Thanks again.

Reply:
Let me know if the fix is okay @RayGuo-ergou. Thanks!